app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt

```diff
@@ -83,11 +88,23 @@ class AddWheelchairAccessBusiness(o: OverpassMapDataAndGeometryApi) : SimpleOver
     override val icon = R.drawable.ic_quest_wheelchair_shop
     override val defaultDisabledMessage = R.string.default_disabled_msg_go_inside
 
-    override fun getTitle(tags: Map<String, String>) = R.string.quest_wheelchairAccess_name_title
+    override fun getTitle(tags: Map<String, String>) = 
+        if (hasFeatureName(tags) && !tags.containsKey("brand"))
+            R.string.quest_wheelchairAccess_name_type_title
+        else
+            R.string.quest_wheelchairAccess_name_title
+    
+    override fun getTitleArgs(tags: Map<String, String>, featureName: Lazy<String?>): Array<String> {
+        val name = tags["name"] ?: tags["brand"] ?: featureName.value
```

Comment:
Why the `?: featureName.value`?

Reply:
Honestly I just copied `getTitleArgs` from AddOpeningHours.kt.
I wasn't sure about it since it is not in in the default `getTitleArgs`, but decided to copy everything from opening hours.

Reply:
The line means
`assign to variable "name" what's the value of the tag "name", if it is not set, what's the value of the tag "brand" and if that is not set, the generic feature name`

Reply:
Then acually the `featureName.value` is useless... (and could give strange questions when occurring twice)
Since the Wheelchair Quest requires a name anyway (in the Overpass query), `tags["name"]` must be set and ` val name = tags["name"]` should be enough.

Reply:
Well, `val name = tags["name"] ?: tags["brand"]` would be enough